app/Models/Traits/HasFields.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: adam
+ * Date: 08/11/2018
+ * Time: 15:29
+ */
+
+namespace App\Models\Traits;
+
+
+use App\Models\Field;
+
+trait HasFields {
+    static function fields(){
+        $form = strtolower(substr( static::class,11));
```

Comment:
I don't understand why we are doing this whole string manipulation? couldn't it just be the classname?

Reply:
you want to save the whole name? sure no problem it was just so it would look good on the table naa mean